source/_components/sensor.ads.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: "ADS Sensor"
+description: "Instructions how to integrate ADS numeric values into Home Assistant."
+date: 2017-10-25 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Sensor
+ha_release: 0.56
+ha_iot_class: "Local Push"
+---
+
+The `ADS` sensor platform allows reading the value of a numeric variable on
+your ADS device. The variable can be of type *INT*, *UINT* or *BYTE*.
+
+To use your ADS device, you first have to set up your [ADS
+hub](/components/ads/) and then add the following to your `configuration.yaml`
+file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: ads
+    adsvar: GVL.temperature
+    unit_of_measurement: '°C'
+    adstype: int
+```
+
+Configuration variables:
+
+- **adsvar** (*Required*): The name of the variable which you want to access on
```

Comment:
Please use the configuration tags for listing configuration variables. For more information:
https://home-assistant.io/developers/documentation/create_page/#configuration